### PR TITLE
feat: General improvements 2

### PR DESCRIPTION
**Pull request by JoaquinLeonArg**

### CODE REVIEW

File: frontend/src/app/[tournamentID]/collection/page.tsx
Comment (fbianco32) [inline frontend/src/app/[tournamentID]/collection/page.tsx:1]: Page crashes when collection filters return no cards

WhatsApp Image 2024-03-15 at 13 31 41_ac21cd70

File: frontend/src/app/[tournamentID]/packs/page.tsx
Comment (fbianco32) [inline frontend/src/app/[tournamentID]/packs/page.tsx:1]: Pack amount can still be negative

WhatsApp Image 2024-03-15 at 13 29 01_3496b72f

Flip all cards has no animation

File: frontend/src/app/[tournamentID]/collection/page.tsx
Comment (fbianco32) [reply frontend/src/app/[tournamentID]/collection/page.tsx:1]: Card amount indicator looks out of place, maybe recolor?

WhatsApp Image 2024-03-15 at 13 29 50_7fe3223a

Comment (JoaquinLeonArg) [reply frontend/src/app/[tournamentID]/collection/page.tsx:1]: 💥

**Pull request merged**